src/redis_cmd.cc

```diff
@@ -1,36 +1,38 @@
+#include "redis_cmd.h"
+
 #include <arpa/inet.h>
 #include <fcntl.h>
+#include <glog/logging.h>
```

Comment:
glog libevent rocksdb are third party libraries, should put after C++ std lib?

Reply:
Yep,u r right. It seems that there still exists some mysterious judgements in the `cpplint`, making the incorrect alignments.
Truely, `glog/logging.h` is the thirdparty lib including in `external`. However, the `cpplint` has made a judgement, as a C system lib, if the `glog/logging.h` was included with `<>`. And If we moved it after the c++ stdlib as the thirdpart the checked results was showed as ``  But, if we could include `glog/logging.h`
`sytax = c++`

Reply:
It seems that there still exists errs in `cpplint-v1.5.5`, i will check it again.

Reply:
so could we put the header files of `glog、libevent and rocksdb` together, and after c system
change from
```c
#include <arpa/inet.h>
#include <fcntl.h>
#include <glog/logging.h>
#include <sys/socket.h>
```
to
```c
#include <fcntl.h>
#include <sys/socket.h>
#include <arpa/inet.h>
#include <glog/logging.h>
```